src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java

```diff
@@ -216,10 +217,12 @@ public ValidateGroupsResult validateGroups(SwitchId switchId, List<GroupEntry> g
     public ValidateLogicalPortsResult validateLogicalPorts(SwitchId switchId, List<LogicalPort> presentLogicalPorts) {
         Map<Integer, LogicalPortInfoEntry> expectedPorts = lagLogicalPortRepository.findBySwitchId(switchId).stream()
                 .map(LogicalPortMapper.INSTANCE::map)
+                .peek(port -> Collections.sort(port.getPhysicalPorts()))
```

Comment:
Note from javadoc for peek() method `This method exists mainly to support debugging`. Maybe we can move sorting inside mapper?